.jenkins/Jenkinsfile

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/groovy
+@Library('jenkins-pipeline@v0.4.5')
+import com.invoca.docker.*;
+pipeline {
+  agent {
+    kubernetes {
+      defaultContainer "ruby"
+      yamlFile ".jenkins/ruby_build_pod.yml"
+    }
+  }
```

Comment:
minor, but a newline after each major section of this file helps with readability.. so here, and after the `environment { }` block, and between each stage.. and above the `void updateGitHubStatus` would be good!